pyproject.toml

```diff
@@ -92,6 +92,7 @@ torchvision = [
   {markers = "sys_platform != 'darwin' or platform_machine != 'x86_64'", version = "^0"},
   {markers = "sys_platform == 'darwin' and platform_machine == 'x86_64'", version = "~0.17.2"}
 ]
+ocrmac = {markers = "sys_platform == 'darwin'", version = "^1.0.0", optional = true}
```

Comment:
This should be moved to the main dependencies above.

Reply:
Thank you for your comment. I have moved the ocrmac dependency to the main dependencies.